content/id/docs/concepts/workloads/pods/pod-preset.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Pod Preset
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+Halaman ini menyajikan sebuah gambaran umum dari `Pod Preset`, yang merupakan objek untuk memasukkan informasi tertentu ke dalam `Pod` pada saat waktu penciptaan. Informasi dapat berupa *secret*, *volume*, *volume mount*, dan variabel *environment*.
```

Comment:
Nit: halaman ini menyajikan gambaran umum tentang...

Reply:
fixed in https://github.com/kubernetes/website/pull/14394/commits/5e8fb53a826b1119b77b923408a4f7a9e64e6415